api/server/cluster_test.go

```diff
@@ -0,0 +1,53 @@
+package server
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/libopenstorage/openstorage/api"
+	client "github.com/libopenstorage/openstorage/api/client/cluster"
+	"github.com/libopenstorage/openstorage/cluster"
+	mockcluster "github.com/libopenstorage/openstorage/cluster/mock"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/golang/mock/gomock"
+	"github.com/kubernetes-csi/csi-test/utils"
+)
+
+func TestServerNodeStatus(t *testing.T) {
+
+	// Save already set value of cluster.Inst to set it back
+	// when we finish the tests by the defer()
+	oldInst := cluster.Inst
+	defer func() {
+		cluster.Inst = oldInst
+	}()
+
+	// Create mock cluster
+	mc := gomock.NewController(&utils.SafeGoroutineTester{})
```

Comment:
Can you separate out the setting of the instance section of the code into a function? All the tests are going to need it.

Reply:
Sure I'll take care of it.